transit_odp/publish/views/timetable/seasonal_services.py

```diff
@@ -64,55 +103,79 @@ def get_form_kwargs(self, step=None):
     @transaction.atomic
     def done(self, form_list, **kwargs):
         all_data = self.get_all_cleaned_data()
-        org_id = self.kwargs["pk1"]
-        SeasonalService.objects.filter(licence__organisation_id=org_id).create(
+        SeasonalService.objects.filter(licence__organisation_id=self.org_id).create(
             licence=all_data["licence"],
             registration_code=all_data["registration_code"],
             start=all_data["start"],
             end=all_data["end"],
         )
+        num_seasonal = SeasonalService.objects.filter(
+            licence__organisation_id=self.org_id
+        ).count()
         return HttpResponseRedirect(
             reverse(
                 "seasonal-service",
-                kwargs={"pk1": org_id},
+                kwargs={"pk1": self.org_id},
                 host=config.hosts.PUBLISH_HOST,
             )
+            + get_table_page(self.page)
         )
 
 
-class SeasonalServiceView(OrgUserViewMixin, SingleTableView):
-    template_name = "publish/seasonal_services/index.html"
+class SeasonalServiceEditDateView(OrgUserViewMixin, BaseUpdateView):
+    template_name = "publish/seasonal_services/edit_date.html"
+    form_class = SeasonalServiceEditDateForm
     model = SeasonalService
-    table_class = SeasonalServiceTable
-    paginate_by = 10
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        self.page = self.request.GET.get("page", None)
+        kwargs.update(
+            {
+                "org_id": self.org_id,
+                "page": self.page,
```

Comment:
no

Reply:
we have to decide for this `page` bug/feature